OpenTabletDriver.UX/Controls/PluginSettingStoreCollectionEditor.cs

```diff
@@ -79,12 +81,15 @@ protected virtual void OnStoreCollectionChanged()
         private void RefreshContent()
         {
             var types = AppInfo.PluginManager.GetChildTypes<TSource>();
+            var sortedTypes = new ReadOnlyCollection<TypeInfo>([.. types.OrderBy(t => t.Name)]);
```

Comment:
Suggestion if you want to apply this:
```suggestion
            var sortedTypes = new ReadOnlyCollection<TypeInfo>([.. types.OrderBy(t => t.GetFriendlyName())]);
```